domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/casetype/CaseTypeServiceImpl.java

```diff
@@ -89,6 +90,15 @@ public boolean caseTypeExistsInAnyJurisdiction(String reference, String jurisdic
             .map(this::addMetadataFields);
     }
 
+    @Transactional
```

Comment:
do we need @Transactional here?

Reply:
Yes all method in this services has been marked as @trasctional